src/setAnyTrustFastConfirmerPrepareTransactionRequest.ts

```diff
@@ -17,7 +17,7 @@ export type SetAnyTrustFastConfirmerPrepareTransactionRequestParams<
   TChain extends Chain | undefined,
 > = {
   publicClient: PublicClient<Transport, TChain>;
-  account: PrivateKeyAccount;
+  account: PrivateKeyAccount | `0x${string}`;
```

Comment:
let's just switch to an account address and drop `PrivateKeyAccount` (and update jsdoc accordingly)

```suggestion
  account: Address;
```